sydent/util/emailutils.py

```diff
@@ -129,6 +125,25 @@ def sendEmail(
         raise EmailSendException() from origException
 
 
+def check_valid_email_address(address: str, allow_description: bool = True) -> None:
```

Comment:
Does this want `allow_description` to default to `False`?

Reply:
Argh, it does. Thank you, I need to slow down and think more carefully.